trillian/ctfe/instance.go

```diff
@@ -133,7 +136,29 @@ func setUpLogInfo(ctx context.Context, opts InstanceOptions) (*logInfo, error) {
 		acceptOnlyCA:    cfg.AcceptOnlyCa,
 		extKeyUsages:    vCfg.KeyUsages,
 	}
+	var err error
+	validationOpts.rejectExtIds, err = parseOIDs(cfg.RejectExtensions)
+	if err != nil {
+		return nil, err
```

Comment:
Add some context to this error, otherwise `setUpLogInfo()` may return an error about not being able to convert a character into an integer without any context about where that character came from.

Reply:
Done